data/constants/keycodes/keycodes_0.0.5_quantum.hjson

```diff
@@ -0,0 +1,25 @@
+{
+    "keycodes": {
+        "0x7C7B": {
+            "group": "quantum",
+            "key": "QK_KEY_CANCELLATION_ON",
+            "aliases": [
+                "KI_ON"
```

Comment:
```suggestion
                "KI_ON"
```
```suggestion
                "KC_ON"
```

Reply:
Yeah wasn't too keen on having `KC` since thats the normal. Im open to changing from `KI` to something else, but I don't think its a good use of `KC` prefix here.

Reply:
Given the other mode is “key exclusion”, what about `KX_` for both?

Reply:
Could always rename it _yet again_… “key scrubbing” with `KS_` perhaps?